src/Node/HTTP/HTTPS.purs

```diff
@@ -0,0 +1,242 @@
+-- | This module defines low-level bindings to the Node HTTPS module.
+
+module Node.HTTP.HTTPS
```

Comment:
`Node.HTTP.Secure` is good (though tbh I don't have any preference for it over `Node.HTTP.HTTPS` since HTTPS is synonymous with Secure--I'm happy to rename it to either).  I don't like `Node.HTTPS`--the vast majority of the methods are shared between the two, so it makes sense to keep them under the same root module--especially since we decided against re-exporting the `Node.HTTP` methods in the `HTTPS` module.